content/en/code_analysis/_index.md

```diff
@@ -78,34 +82,63 @@ Manage the full list of libraries detected by SCA that have imported into your c
 {{% /tab %}}
 {{< /tabs >}}
 
-With Static Analysis, you can receive automated feedback on poor coding practices and security vulnerabilities on the code you write directly in an IDE such as [VS Code][3] or [IntelliJ & PyCharm][4], and in your [pull requests on GitHub][5]. 
+With Static Analysis, you can receive automated feedback on poor coding practices and security vulnerabilities on the code you write [directly in an IDE][11] such as [VS Code][3] or [IntelliJ & PyCharm][4], and in your [pull requests on GitHub][5]. 
+
+{{< img src="developers/ide_plugins/vscode/static-analysis-issue.png" alt="A Static Analysis result in Visual Studio Code" style="width:100%;">}}
 
 ## Set up Code Analysis on your repository
 
-Click **+ Add a Repository** on the [**Code Analysis Repositories** page][9] and select the relevant programming languages to add Code Analysis to your project. Datadog provides out-of-the-box rulesets for the following languages:
+Click **+ Add a Repository** on the [**Code Analysis Repositories** page][9] and choose to run the scans directly in Datadog or in your CI pipelines.
+
+{{< tabs >}}
+{{% tab "Datadog" %}}
 
-{{< partial name="code_analysis/languages-getting-started.html" >}}
+<div class="alert alert-warning">Datadog-hosted scans are supported by Software Composition Analysis (SCA) and GitHub repositories only. To enable Static Analysis or if you are not using a GitHub repository, run scans in your CI pipelines instead.</div>
```

Comment:
```suggestion
<div class="alert alert-warning">Datadog-hosted scans are supported by Software Composition Analysis (SCA) and GitHub repositories only. To enable Static Analysis or use a different CI provider, run scans in your CI pipelines instead.</div>
```

Reply:
Not sure if this is the most accurate way to describe alternative repos other than Github.